### PR TITLE
Updates from bedrock

### PR DESCRIPTION
**Pull request by MozmarRobot**
From file changes in https://github.com/mozilla/bedrock/commit/84d2eef22d2ec4ac6b86eb33b45e9df00b3639a1

### CODE REVIEW

**Review by peiying2 — CHANGES_REQUESTED** [review]
The revised content contains straight quotes, not curly quotes. Otherwise, it looks good.

**Comment by peiying2** [comment]
@craigcook Not sure who is working on this. The changes contain a few straight quotes.

File: en/products/vpn/more/ip-address.ftl [hunk 1]
```diff
@@ -45,6 +45,13 @@ vpn-ip-address-you-dont-need = You don’t need to hide your IP address, but the
...
+#   $vpn (url)- https://www.mozilla.org/products/vpn/more/what-is-a-vpn/
+#   $mozvpn (url) - https://www.mozilla.org/products/vpn/
+#   $countries (number) - number of countries where Mozilla VPN has servers, e.g. "30". The + after indicates the number may be higher, where "30+" means "30 or more".
+vpn-ip-address-a-vpn-is-v2 = A VPN is a way to hide your IP address. <a href="{ $vpn }">When you use a VPN</a>, your external IP address will be coming from the VPN server’s external IP, rather than your location’s external IP address. So if your connecting VPN server is located in California, your external IP will look like it’s connected from California, no matter where you actually are. Plus, your online activity is sent over an encrypted, secure connection to your VPN server, giving you additional security and privacy. <a href="{ $mozvpn }">{ -brand-name-mozilla-vpn }</a> is one way to hide your IP address. We don't keep your network activity logs, and we don't partner with third parties who build profiles of what you do online. We offer full-device protection for up to five devices with servers in { $countries }+ countries, you can connect to anywhere, from anywhere.
```
Comment (peiying2) [inline en/products/vpn/more/ip-address.ftl:52]: Change straight quotes to curly quotes.

File: en/products/vpn/more/what-is-a-vpn.ftl [hunk 2]
```diff
@@ -59,6 +59,11 @@ vpn-what-is-are-vpns-only = Are VPNs only for computers?
 vpn-what-is-vpns-can-be = VPNs can be used to protect all your devices — computers, laptops, tablets and phones. If you access the internet through stand-alone apps beyond the browser, a VPN is a good choice for protecting your privacy and security on any device. You will need to install a VPN on each device for it to be effective. <a href="{ $mozvpn }">{ -brand-name-mozilla-vpn }</a> offers an unlimited connection for up to five devices when you connect to the internet from any app or browser.
 vpn-what-is-mozilla-vpn-fast = { -brand-name-mozilla-vpn }: Fast, secure, trustworthy
 
+# Variables
+#   $mozvpn (url) - https://www.mozilla.org/products/vpn/
+vpn-what-is-mozilla-vpn-is-a-v2 = <a href="{ $mozvpn }">{ -brand-name-mozilla-vpn }</a> is a service that you can trust to keep your connection to the internet safe on all your devices. We don't keep your network activity logs, and we don't partner with third parties who build profiles of what you do online. In a world where unpredictability has become the new normal…
```
Comment (peiying2) [inline en/products/vpn/more/what-is-a-vpn.ftl:64]: Change straight quotes to curly quotes.

File: en/firefox/browsers/mobile/focus.ftl [hunk 3]
```diff
@@ -20,20 +20,36 @@ mobile-focus-send-a-download-link-to-your = Send a download link to your phone.
...
+mobile-focus-delete-your-history = Delete your history
+
 mobile-focus-easily-erase-your = Easily erase your history, passwords and cookies, so unwanted ads don’t follow you around online. Just tap the erase button on the search field and it’s gone.
 
 mobile-focus-take-private-mode = Take private mode to the next level
+
+# Outdated string
 mobile-focus-private-browsing-on = “Private browsing” on most browsers isn’t really all that private. { -brand-name-firefox-focus } is next-level, always on privacy and it’s backed by { -brand-name-mozilla }, the non-profit that fights for your rights on the web.
 
+mobile-focus-private-browsing-on-v2 = “Private browsing” on most browsers isn’t really all that private. { -brand-name-firefox-focus } is next-level, private by default and it’s backed by { -brand-name-mozilla }, the non-profit that fights for your rights on the web.
```
Comment (peiying2) [inline en/firefox/browsers/mobile/focus.ftl:35]: This sentence seems missing a word, `{ -brand-name-firefox-focus } is next-level,`. Next level what??

Comment (pmac) [reply en/firefox/browsers/mobile/focus.ftl:35]: That's an americanism I guess. 

> Something or someone on a level above all the rest of the world. The person on the 'NEXT LEVEL' usually thinks differently from everyone, hence why he is known as NEXT LEVEL.
> 
> A thing can also be called NEXT LEVEL. Anything above average basically.

From https://www.urbandictionary.com/define.php?term=next%20level

Comment (peiying2) [reply en/firefox/browsers/mobile/focus.ftl:35]: Thanks for the explanation. I understand `next level`. When it's hyphenated, I expect there is a word after this.

Comment (pmac) [reply en/firefox/browsers/mobile/focus.ftl:35]: good point. I think it being hyphenated here is a mistake.

Comment (craigcook) [reply en/firefox/browsers/mobile/focus.ftl:35]: I think that may be deliberate, it's just very "west coast American." Maybe we need a simpler version for canonical English and if they really want to keep "next-level" we can do it in the en-US file. I'll ask.

Comment (craigcook) [reply en/firefox/browsers/mobile/focus.ftl:35]: We're changing it to "{ -brand-name-firefox-focus } offers next-level privacy by default" which is MUCH clearer.

Comment (peiying2) [reply en/firefox/browsers/mobile/focus.ftl:35]: Yeah, the comma threw things off. This is the proper expressions.

**Review by peiying2 — APPROVED** [review]
Thank you for the revision of the string  and other fixes. LGTM!

**Pull request merged**